### PR TITLE
Light cover support

### PR DESCRIPTION
**Pull request by KeitIG**
This adds basic album cover support:

- The image is stored as base64 in the database
- Covers are track-specific, not album-specific

### CODE REVIEW

File: src/js/components/Header/PlayingBar.react.js [hunk 1]
```diff
@@ -104,6 +114,13 @@ export default class PlayingBar extends Component {
         clearInterval(this.timer);
     }
 
+    componentWillReceiveProps(nextProps) {
+
+        const trackPlaying =  nextProps.queue[nextProps.queueCursor];
+
+        this.coverBase64 = trackPlaying && trackPlaying.cover ? `data:image/${trackPlaying.cover.format};base64,${trackPlaying.cover.data}` : '';
```
Comment (YurySolovyov) [inline src/js/components/Header/PlayingBar.react.js]: these 2 lines and ones on 43+ look the same, mind adding a method?

File: src/js/components/Header/TrackCover.react.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    constructor(props) {
+
+        super(props);
+    }
+
+    render() {
+
+        if(this.props.cover) return <div className='cover' style={ { backgroundImage: `url(${this.props.cover})` } }></div>;
```
Comment (YurySolovyov) [inline src/js/components/Header/TrackCover.react.js]: at least styles should be in a separate method

File: src/styles/_header.scss [hunk 3]
```diff
@@ -115,122 +116,149 @@ header {
...
-                justify-content: flex-start;
-                min-width: 100px;
+                justify-content: center;
 
-                .button {
-                    border: 0;
-                    background: transparent;
-                    font-size: 18px;
-                    -webkit-app-region: no-drag;
+                .pf {
```
Comment (YurySolovyov) [inline src/styles/_header.scss]: maybe rename it to something more descriptive?

File: src/js/components/Header/TrackCover.react.js (hunk 2 above)
Comment (KeitIG) [inline src/js/components/Header/TrackCover.react.js]: Why ? There are other inline styles like that in the app.

File: src/styles/_header.scss (hunk 3 above)
Comment (KeitIG) [inline src/styles/_header.scss]: some old code (`.pf` was for "player font"). I'll change this.

File: src/js/components/Header/TrackCover.react.js (hunk 2 above)
Comment (YurySolovyov) [inline src/js/components/Header/TrackCover.react.js]: I just don't like how long this line is, maybe you can wrap it into 2 lines?

File: src/styles/_header.scss [hunk 4]
```diff
@@ -115,122 +116,149 @@ header {
...
-            display: flex;
-            align-items: center;
-            justify-content: center;
+        .now-playing-cover {
 
-            .pf {
+            .cover {
                 display: flex;
-                align-items: center;
+                align-items: flex-end;
```
Comment (KeitIG) [inline src/styles/_header.scss]: Auto-reminder: this is wrong.

**Comment by KeitIG** [comment]
I get we're good to go. It's basic, but it does the job.

**Pull request merged**